tests/test_agrregation_region_aggrfuncs.py

```diff
@@ -0,0 +1,241 @@
+import pyam
```

Comment:
typo in filename; how about just `test_region_aggregation`?

Reply:
w.avg - weighted averages are handled by `weighted_average_region` func.
e.g.:
`    df.weighted_average_region('Price', append=True, weight='Primary Energy')`